<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MasternodeManager</class>
 <widget class="QWidget" name="MasternodeManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>723</width>
    <height>457</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background: #fcfcfc;</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="styleSheet">
        <string notr="true">background-color:#fcfcfc;</string>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <property name="usesScrollButtons">
        <bool>false</bool>
       </property>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>MY MASTERNODES</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="createButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#createButton {
background-color: #00A4B3;
}
#createButton:hover{ 
    background-color: #00757F;
}</string>
               </property>
               <property name="text">
                <string>&amp;Create...</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="copyAddressButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#copyAddressButton:hover {
    background: #00A4B3;
    color: #fff;
}</string>
               </property>
               <property name="text">
                <string>Copy &amp;Address</string>
               </property>
               <property name="default">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableWidget" name="tableWidget_2">
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="alternatingRowColors">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <property name="showGrid">
              <bool>false</bool>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>Alias</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>IP/Onion</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Status</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Collateral Address</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="startButton">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#startButton {background-color: #00A4B3;}
#startButton:hover{ 
    background-color: #00757F;
}
#startButton:disabled {
    background-color: #cccccc;
}</string>
               </property>
               <property name="text">
                <string>&amp;Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stopButton">
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="styleSheet">
                <string notr="true">#stopButton:hover {
    background: #00A4B3;
    color: #fff;
}</string>
               </property>
               <property name="text">
                <string>Sto&amp;p</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QComboBox" name="actionsBox">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">QComboBox {
	width: 120px;
}

QListView {
    border: 2px solid #E9E9E9;
	 padding: 10px;
	width: 120px;
}

QComboBox::item {
    color:#00A4B3;
	 text-transform: uppercase;
}</string>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string>Edit</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Get config</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Remove</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Actions</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tab">
        <attribute name="title">
         <string>NETWORK</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Masternodes Active:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="countLabel">
             <property name="text">
              <string>0</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <widget class="QTableWidget" name="tableWidget">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="alternatingRowColors">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>60</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Rank</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Active time</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Pubkey</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
